packages/apfelxx/variables.sh

```diff
@@ -0,0 +1,4 @@
+VERSION=4.7.0
+URL="https://github.com/vbertone/apfelxx/archive/refs/tags/$VERSION.tar.gz"
+TARBALL="apfelxx-$VERSION.tar.gz"
+SRC_DIR="apfelxx-$VERSION"
```

Comment:
```suggestion
URL="https://github.com/vbertone/apfelxx.git"
SRC_DIR="apfelxx"
```

Reply:
Then please use `https://github.com/vbertone/apfelxx/tarball/resummation-scales`.

```suggestion
VERSION="resummation-scales"
URL="https://github.com/vbertone/apfelxx/tarball/$VERSION"
TARBALL="apfelxx-$VERSION.tar.gz"
SRC_DIR="apfelxx-$VERSION"
```

Avoid Git as much as possible (and for as long as you want to download from GitHub, that's practically always possible).